mule-bpm-module/README.md

```diff
@@ -74,6 +74,14 @@ The core BPM engine implementation used by this Mule BPM Module. It is not curre
 
 ## Features
 
+### UniqueDeploymentName
+
+When 'duplicateDeploymentFiltering' is set to true and 'uniqueDeploymentName' is provided, 
```

Comment:
Add description of result of setting this true: prevents deploying same process and form definitions, if there are no changes

Reply:
[888b027](https://github.com/alfameCom/mule-bpm-module/commit/888b027bdaec556141d1e0863c9d29ee758a8b19)